.isort.cfg

```diff
@@ -4,4 +4,5 @@ force_sort_within_sections=true
 ignore_whitespace=true
 known_third_party=apache_beam,backports.functools_lru_cache,browsermobproxy,cloudstorage,contextlib2,elasticsearch,firebase_admin,google.api_core,google.appengine,google.cloud,google.protobuf,mapreduce,mock,mutagen,pipeline,pkg_resources,psutil,pylatexenc,pylint,requests,requests_mock,selenium,six,skulpt,typing,webapp2,webapp2_extras,webtest,yaml
 line_length=80
+multi_line_output=7
```

Comment:
No, their documentation is not 100 % becuase this also disables isort errors when some line is longer than 80 chars. But since I refactored the code anyway this is no longer needed.